src/components/EventListCard/Modal/updateLogic.ts

```
@@ -0,0 +1,224 @@
+
+      if (!isRecurringInstance) {
+        const result = await updateStandaloneEvent({
+          variables: { input: updateInput },
+        });
+        data = result.data;
+      } else {
+        switch (updateOption) {
+          case 'single':
+            const singleResult = await updateSingleRecurringEventInstance({
+              variables: { input: updateInput },
+            });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)